support/ebpf/types.h

```diff
@@ -887,4 +940,13 @@ typedef struct ApmIntProcInfo {
   u64 tls_offset;
 } ApmIntProcInfo;
 
-#endif
+typedef struct GoCustomLabelsOffsets {
+  u32 m_offset;
+  u32 curg;
+  u32 labels;
+  u32 hmap_count;
+  u32 hmap_log2_bucket_count;
+  u32 hmap_buckets;
+} GoCustomLabelsOffsets;
```

Comment:
```suggestion
} GoLabelProcInfo;
```
So it aligns with the potentially updated bpf map, and indicates its per-proc data. We maybe want to add also the memoized `m` ptr here?